spec/support/examples.rb

```diff
@@ -274,6 +274,28 @@ def self.xml
     end
   end
 
+  class CustomExampleEmptyNodeWithAttribute < ExampleBase
+    def self.json
+      <<~JSON
+        {
+          "alice": {
+            "bob": {
+              "@foo": "bar"
+            }
+          }
+        }
+      JSON
+    end
+
+    def self.xml
+      <<-XML
+        <alice>
+			    <bob foo="bar"/>
+		    </alice>
```

Comment:
Here's some weird formatting:

<img width="410" alt="Bildschirmfoto 2023-07-14 um 09 25 32 1" src="https://github.com/openHPI/dachsfisch/assets/7300329/4b1d8721-d0ff-4857-ac33-bedc930f823e">

Would you mind fixing that? 🙏

Reply:
done